### DISCUSSION TITLE
loop_type support

### DISCUSSION THREAD

**Discussion by AlexMitchellMus** [sfztools/sfizz#1212] [Requests]
Support Loop_type opcode.

Allows sample looping to be:
• forward
• backward
• alternate

https://sfzformat.com/opcodes/loop_type.html

**Comment by kinwie** [comment]
This is supported in DropZone, Dimension and Rapture.
Sforzando parsed this silently and just set it to forward only.

**Comment by mathieugarcia** [comment]
Yup, I would love to see this one implemented.
